include/mumble/Message.hpp

```diff
@@ -67,6 +67,72 @@ struct Message {
 	Timestamp timestamp;
 };
 
+namespace udp {
+	struct Message : public mumble::Message {
+		struct Audio;
+		struct Ping;
+
+		enum class Type : uint8_t { Audio, Ping, Unknown = std::numeric_limits< uint8_t >::max() };
+
+		Message()          = default;
+		virtual ~Message() = default;
+
+		Protocol protocol() const { return Protocol::UDP; }
```

Comment:
```suggestion
		Protocol protocol() const override { return Protocol::UDP; }
```
We should make use of this keyword as it helps avoiding accidentally not override due to a slightly off signature.

Reply:
Indeed, it wasn't omitted on purpose.